Tests/conf.json

```diff
@@ -1871,6 +1871,10 @@
             "instance_names": "es_v6",
             "playbookID": "Elasticsearch_v2_test-v6"
         }
+	{
+            "integrations": "ARIA Packet Intelligence",
```

Comment:
we can remove this, not needed anymore

Reply:
Okay

Reply:
I removed froom the conf.json, then the CI will report a failure: `#!/bin/bash -eo pipefail
[ -n "${NIGHTLY}" ] && IS_NIGHTLY=true || IS_NIGHTLY=false
python ./Tests/scripts/configure_tests.py -n $IS_NIGHTLY
Starting creation of test filter file
Getting changed files from the branch: pull/4330
The following ids are affected due to the changes you made:

There is no test-playbook that checks the changes you've done, please make sure you write one.
No filter configured, running all tests
Creating filter_file.txt
Failed test configuration. See previous errors.`

Reply:
committed two changes that seem to fix it

Reply:
Yeah, that fix it in the CI. Thanks.